<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>350</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Resource/resource.qrc">:/Logos/Shelby-Logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>10</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Resource/resource.qrc">:/Logos/fians.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Resource/resource.qrc">:/Logos/uatlogopng.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Img_alan">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Resource/resource.qrc">:/Integrantes/Img_alan.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_alan">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(219, 237, 255);</string>
    </property>
    <property name="text">
     <string>Alan</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_equipo">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>300</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(219, 237, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
    border: 2.5px solid balck;
    color: black;</string>
    </property>
    <property name="text">
     <string>Equipo: </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_diego">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>250</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(219, 237, 255);</string>
    </property>
    <property name="text">
     <string>Diego</string>
    </property>
   </widget>
   <widget class="QLabel" name="Img_diego">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Resource/resource.qrc">:/Integrantes/Img_diego.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_joseph">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>250</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(219, 237, 255);</string>
    </property>
    <property name="text">
     <string>Joseph</string>
    </property>
   </widget>
   <widget class="QLabel" name="Img_joseph">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Resource/resource.qrc">:/Integrantes/Img_joseph.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
    border: 2.5px solid balck;
    color: black;</string>
    </property>
    <property name="text">
     <string>Escoge integrantes...</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_emma">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>250</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(219, 237, 255);</string>
    </property>
    <property name="text">
     <string>Emmanuel</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_isaac">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>250</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(219, 237, 255);</string>
    </property>
    <property name="text">
     <string>Isaac</string>
    </property>
   </widget>
   <widget class="QLabel" name="Img_joseph_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>140</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Resource/resource.qrc">:/Integrantes/emmanuel.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Img_joseph_3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>140</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Resource/resource.qrc">:/Integrantes/duck.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1091</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../Resource/resource.qrc"/>
  <include location="../../Resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
